llvm/test/CodeGen/AMDGPU/insert-skips-flat-vmem-ds.mir

```diff
@@ -1,95 +0,0 @@
-# NOTE: Assertions have been autogenerated by utils/update_mir_test_checks.py
-# RUN: llc -mtriple=amdgcn -mcpu=polaris10 -run-pass si-pre-emit-peephole -amdgpu-skip-threshold=1 -verify-machineinstrs  %s -o - | FileCheck %s
```

Comment:
Keep this?

Reply:
This test was used to check that `SIPreEmitPeephole` did not optimize vmem memory accesses; but now they are handled.

I've added a commit before the main commit of this patch, to be submitted separately.
I restored it, renamed it, and added some branch weights such that the transformation is profitable.